Firebase.agent.lib.nut

```diff
@@ -592,20 +636,32 @@ class Firebase {
     }
 
     // process the http response accordingly
-    function _processResponse (request,callback) {
+    function _invokeCallback(request, callback) {
```

Comment:
and I'd rename that one too ) anyway this one doesn't invoke the callback, it sends the request